docs/guides/docker_image_with_your_release.md

```diff
@@ -0,0 +1,74 @@
+# Build a docker image with your release
+
+Now that we have seen how to build a release, let's build one in a docker image. This is necessary for a deployment with docker swarm or kubernetes for example.
+
+A prerequisite step to this, is to initialize your release (`mix release.init`) and configure your [vm.args](/files/vm.args).
```

Comment:
I would make this a callout, e.g.

```
!!! info
    Make sure you have initialized Distillery in your project with `mix release.init` and configured your `vm.args` file as mentioned [here](../config/vm.args.md)
```

Do they need to configure `vm.args` though? If it's not actually a prerequisite, I would leave that out.